bin/run.sh

```diff
@@ -66,14 +66,17 @@ run_tests() {
     # Run tests and pipe output to results.out file.
 
     local slug="$1"
-    local test_file=${slug//-/_}_test.sh
+    local test_file=${slug//-/_}.bats
     local solution_dir="$2"
     local output_file="$3"
 
     echo "Running tests."
 
     cd "$solution_dir"
 
+    # test scripts may be (old) xxxx_test.sh
+    [[ -f "$test_file" ]] || test_file=${slug//-/_}_test.sh
```

Comment:
Would it be cleaner to have the old and new names set adjacent to each other?

Reply:
Not for me, but I did it...